x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/utils.tsx

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+export const getConnectedBadgeAriaLabel = (connectedCount: number) =>
+  i18n.translate('xpack.enterpriseSearch.connectorStats.connectedBadgeAriaLabel', {
+    defaultMessage: '{number} Connected connectors',
+    values: {
+      number: connectedCount,
+    },
+  });
+export const getConnectedConnectorsBadgeLabel = (connectedCount: number) => (
+  <FormattedMessage
+    id="xpack.enterpriseSearch.connectorStats.connectedBadgeLabel"
+    defaultMessage="{number} Connected"
+    values={{
+      number: connectedCount,
+    }}
+  />
+);
+export const getConnectedConnectorsTooltipContent = (
+  connectedCount: number,
+  isCrawler: boolean
+) => (
+  <EuiText size="xs">
+    {!isCrawler ? (
+      <FormattedMessage
+        id="xpack.enterpriseSearch.connectorStats.connectedTooltip"
+        defaultMessage="{connectedCount} {completeConnectorsText} - Number of connectors which are complete in the last 30 minutes"
+        values={{
+          completeConnectorsText: (
+            <b>
+              {i18n.translate('xpack.enterpriseSearch.connectorStats.connectorTooltipConnected', {
+                defaultMessage: `Connected connectors`,
+              })}
+            </b>
+          ),
+          connectedCount: <b>{connectedCount}</b>,
+        }}
+      />
+    ) : (
+      <FormattedMessage
+        id="xpack.enterpriseSearch.connectorStats.connectedCrawlerTooltip"
+        defaultMessage="{connectedCount} {completeConnectorsText} - Number of crawlers that are configured and connected."
+        values={{
+          completeConnectorsText: (
+            <b>
+              {i18n.translate('xpack.enterpriseSearch.connectorStats.crawlerTooltipConnected', {
+                defaultMessage: `Connected crawlers`,
+              })}
+            </b>
+          ),
+          connectedCount: <b>{connectedCount}</b>,
+        }}
+      />
+    )}
+  </EuiText>
+);
+export const getIncompleteConnectorsTooltip = (incompleteCount: number, isCrawler: boolean) => (
+  <EuiText size="xs">
+    {!isCrawler ? (
+      <FormattedMessage
+        id="xpack.enterpriseSearch.connectorStats.incompleteTooltip"
+        defaultMessage="{incompleteCount} {incompleteConnectorsText} - Number of connectors which are not ready to sync (i.e. the connector is not completely configured), or there’s no backend connector running which knows how to sync it."
```

Comment:
```suggestion
        defaultMessage="{incompleteCount} {incompleteConnectorsText} - Number of connectors whose configuration is incomplete. Syncs won't be possible until they are fully configured."
```

Reply:
We should involve "no connector running" as well.

Reply:
I tweaked the suggestion, let me know if that works